docs/standard/native-interop/cominterop.md

```diff
@@ -0,0 +1,16 @@
+---
+title: COM Interop in .NET
+description: Learn how to interoperate with COM libraries in .NET.
+author: jkoritzinsky
+ms.author: jekoritz
+ms.date: 07/11/2019
+---
+
+# COM Interop in .NET
+
+The Component Object Model (COM) lets an object expose its functionality to other components and to host applications. To help enable users to interoperate with their existing code bases, the .NET Framework has always provided strong support for interoperating with COM libraries. In .NET Core 3.0, a large portion of this support has been added to .NET Core on Windows. The documentation here will explain how the common COM interop techonologies work and how you can utilize them to interoperate with your existing COM libraries.
```

Comment:
```suggestion
The Component Object Model (COM) lets an object expose its functionality to other components and to host applications on Windows platforms. To help enable users to interoperate with their existing code bases, the .NET Framework has always provided strong support for interoperating with COM libraries. In .NET Core 3.0, a large portion of this support has been added to .NET Core on Windows. The documentation here will explain how the common COM interop techonologies work and how you can utilize them to interoperate with your existing COM libraries.
```
I thought that making it clear that this applies to Windows only in the first sentence is important.